web/app/js/components/Navigation.test.jsx

```diff
@@ -122,3 +127,120 @@ describe('Navigation', () => {
     });
   });
 });
+
+describe('Namespace Select Button', () => {
+  it('renders All Namespaces text', () => {
```

Comment:
I would add a little more detail to this description: `displays All Namespaces as button text if the selected namespace is _all`